application/src/main/java/org/thingsboard/server/service/cf/ctx/state/TsRollingArgumentEntry.java

```diff
@@ -118,8 +118,8 @@ private void addTsRecord(Long ts, KvEntry value) {
             }
             cleanupExpiredRecords();
         } catch (Exception e) {
-            log.warn("Time series rolling arguments supports only numeric values.");
-//            throw new IllegalArgumentException("Time series rolling arguments supports only numeric values.");
+            tsRecords.put(ts, Double.NaN);
+            log.warn("Invalid value '{}' for time series rolling arguments. Only numeric values are supported.", value.getValue());
```

Comment:
Shouldn't we throw the error and create debug event?
Anyway, WARN log level is too much, debug is fine